drivers/hmis/lib/tasks/one_time_migration_20241111.rake

```diff
@@ -0,0 +1,25 @@
+desc 'One time data migration to populate created_by and updated_by on Custom Assessments'
+# rails driver:hmis:migrate_assessments_20241111
+task migrate_assessments_20241111: [:environment] do
+  assessments = Hmis::Hud::CustomAssessment.joins(:versions).distinct
+  total_records = assessments.count
+
+  assessments.find_each.with_index(1) do |custom_assessment, index|
+    versions = custom_assessment.versions.order(created_at: :desc)
+    update_version = versions.first
+    create_version = versions.find_by(event: "create")
+
+    if !create_version || !update_version
```

Comment:
I'm not 100% confident, but that was my intended logic. My thinking was just that if we have versions for this object but no version marked "create," then some funny business is going on and we should investigate.

Now that I'm looking at it again, `update_version` will never be nil since we stop processing if versions is an empty arr. So we can just log something useful if `create_version` is nil and continue processing otherwise.

Reply:
Oh, that sounds right to me. Ok, I'll remove the unneeded log.